deployment/hasura/migrations/Aerie/8_procedural_scheduling/down.sql

```diff
@@ -0,0 +1,46 @@
+begin;
+
+drop trigger if exists "notify_hasura_refreshSchedulingProcedureParameterTypes_INSERT" on scheduler.scheduling_goal_definition;
+drop trigger if exists "notify_hasura_refreshSchedulingProcedureParameterTypes_UPDATE" on scheduler.scheduling_goal_definition;
```

Comment:
2 minor questions:

1) Why is this wrapped in a commit?
2) Does Hasura not clean up these triggers when the metadata changes?

Reply:
1. Not a particular reason for this migration, just a generate best practice for atomic migrations.
2. Hasura doesn't seem to clean them up, but I've added these triggers as exceptions for `pgcmp`, and will remove them from this migration, since they weren't satisfying `pgcmp`. We had to do the same thing for e.g. `refreshActivityTypes`

Reply:
1) The migrations are already run atomically via Hasura. That said, the transaction isn't a blocker, I was commenting on it because it wasn't consistent (though don't mind changing this part of the practice).
2) If Hasura isn't cleaning them up, then I can go either way on us doing so, although if we are then we're banking on Hasura never renaming them.

Reply:
1. Gotcha, I can remove this transaction then if Hasura runs its own; I had them in there while initially testing using `psql -f up.sql`.
2. I'll remove these `drop trigger` statements, and keep the exceptions for `pgcmp`, since it looks like that's what we had to do with the old `exceptions_merlin_down` before the DB merge